app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/settings/autofill/AutoFillScreen.kt

```diff
@@ -79,6 +82,12 @@ fun AutoFillScreen(
             AutoFillEvent.NavigateToBlockAutoFill -> {
                 onNavigateToBlockAutoFillScreen()
             }
+
+            AutoFillEvent.NavigateToSettings -> {
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.UPSIDE_DOWN_CAKE) {
```

Comment:
I agree, it's absolutely not necessary, except to fix a compiler error, since the `CredentialManager.create(context).createSettingsPendingIntent()` method isn't available pre-Android 14